product-list.html

```diff
@@ -0,0 +1,333 @@
+                    </div>
+                </div>
+            </div>                
+        </div>
+
+        <div class="ecom-card vertical-card">
+            <div class="ecom-card-img">
+                <img src="asset/product1-spacTrend.webp" alt="" srcset="">
+            </div>
+            <div class="ecom-details">
+                <div class="ecom-card-details">
+                    <div>
+                        <h2 class="ecom-card-title">Roadster</h2>
+                        <h4 class="ecom-card-subtitle">Men Printed Pullover</h4>
+                    </div>
+                    <div class="product-prices">
+                        <div>
+                            <h4 class="final-price">Rs. 999</h4>
+                            <h4 class="original-price">Rs. 999</h4>
+                        </div>
+                        <h4 class="product-discount">( 50% OFF )</h4>
+                    </div>
+                </div>
+                <div class="card-actions">
+                    <div class="card-action-buttons">
+                        <button class="btn text-button">Add to cart</button>
+                    </div>
+                    <div class="card-action-icons">
+                        <button class="btn icon-button" title="Add to favorites"><i class="fas fa-heart"></i></button>
+                    </div>
+                </div>
+            </div>                
+        </div>
+
+        <div class="ecom-card vertical-card">
+            <div class="ecom-card-img">
+                <img src="asset/product2-spacTrend.webp" alt="" srcset="">
+            </div>
+            <div class="ecom-details">
+                <div class="ecom-card-details">
+                    <div>
+                        <h2 class="ecom-card-title">Roadster</h2>
+                        <h4 class="ecom-card-subtitle">Men Printed Pullover</h4>
+                    </div>
+                    <div class="product-prices">
+                        <div>
+                            <h4 class="final-price">Rs. 999</h4>
+                            <h4 class="original-price">Rs. 999</h4>
+                        </div>
+                        <h4 class="product-discount">( 50% OFF )</h4>
+                    </div>
+                </div>
+                <div class="card-actions">
+                    <div class="card-action-buttons">
+                        <button class="btn text-button">Add to cart</button>
+                    </div>
+                    <div class="card-action-icons">
+                        <button class="btn icon-button" title="Add to favorites"><i class="fas fa-heart"></i></button>
+                    </div>
+                </div>
+            </div>                
+        </div>
+
+        <div class="ecom-card vertical-card">
+            <div class="ecom-card-img">
+                <img src="asset/product3-spacTrend.webp" alt="" srcset="">
```

Comment:
try to put alt value Because
Alt text is also important when browsers are unable to render a page properly, meaning the web page does not display the images. In place of images the website will return the alt attributes instead, describing what image should be showing.